install-and-configure/install/multi-cluster/federated-etl/aggregator.md

```diff
@@ -138,3 +142,33 @@ As `ETL_DAILY_STORE_DURATION_DAYS` increases, the amount of time it will take
 for Aggregator to make data available will increase. You can run `kubectl get
 pods` and ensure the `aggregator` pod is running, but should still wait for all
 data to be ingested.
+
+## Troubleshooting Aggregator
+
+### Resetting Aggregator StatefulSet data
+
+When deploying the Aggregator as a StatefulSet, it is possible to perform a reset of the Aggregator data. The Aggregator itself doesn't store any data, and relies on object storage. As such, a reset involves removing that Aggregator's local storage, and allowing it to re-ingest data from the object store. The procedure is as follows:
+
+1. Scale down the Aggregator StatefulSet to 0
+2. When the Aggregator pod is gone, delete the `aggregator-db-storage-xxx-0` PVC
+3. Scale the Aggregator StatefulSet back to 1. This will re-create the PVC, empty.
+4. Wait for Kubecost to re-ingest data from the object store. This could take from several minutes to several hours, depending on your data size and retention settings.
+
+### Aggregator not displaying any data to frontend after several hours
+
+One reason you may not see data in the frontend yet is because the Aggregator is processing all your ETL files in the federated store bucket into SQL tables.
+
+If you are seeing a lot of the following logs, it could be an indicator that your `.Values.kubecostAggregator.env.DB_BUCKET_REFRESH_INTERVAL` may be set too low, causing the Aggregator to continuously restart its data ingestion process:
+
+```txt
+INF asset worker context cancelled: context canceled
+INF allocation worker context cancelled: context canceled
+```
+
```

Comment:
Maybe add something like this:

The default value of `DB_BUCKET_REFRESH_INTERVAL` is tuned for the first-install experience of Kubecost, meaning it prioritizes small data becoming available more quickly. As your data scale and retention increase and particularly once multiple clusters are set up, the value should be increased.

Reply:
Oh that's good context! I've added that as part of the description of the env var.